acknowledgements.txt

```diff
@@ -0,0 +1,28 @@
+The following institutions, programs, and missions have provided funding
+for pysat development.
+
+Institutions
+------------
+Cosmic Studio
+DARPA Defense Sciences Office
+Office of Naval Research (ONR)
+National Aeronautics and Space Administration (NASA)
+National Oceanic and Atmospheric Administration (NOAA)
+National Science Foundation (NSF)
+
+Missions
+--------
+NOAA Constellation Observing System for Meteorology Ionosphere and Climate (COSMIC-2)
+NASA Ionospheric Connections Explorer (ICON)
+NASA Scintillation Observations and Response of the Ionosphere to Electrodynamics (SORTIE)
+NASA Scintillation Prediction Observations Research Task (SPORT)
+
+Programs
+--------
+NSF 125908, AGS-1651393
+NASA NNX10AT02G, NNH20ZDA001N-LWS, 80NSSC18K120, and 80NSSC21M0180
+NASA Space Precipitation Impacts (SPI) project
+Naval Research Laboratory N00173191G016 and N0017322P0744
+
+
```

Comment:
I'm unsure when to use ONR vs NRL. I believe my contracts were NRL but normally I see folks using ONR.